modules/bundles/get_started.ipynb

```diff
@@ -164,8 +164,6 @@
     "        \"$import os\",\n",
```

Comment:
<p><code></code><code></p><p>A MONAI bundle usually includes the stored weights of a model, TorchScript model, JSON files which include configs and metadata about the model, information for constructing training, inference, and post-processing transform sequences, plain-text description, legal information, and other data the model creator wishes to include.</p><p>For more information about MONAI bundles read the description: <a href="https://docs.monai.io/en/latest/bundle_intro.html" target="_blank">https://docs.monai.io/en/latest/bundle_intro.html</a>.</p><p>This notebook is a step-by-step tutorial to help get started to develop a bundle package, which contains a config file to construct the training pipeline and also has a <code>metadata.json</code> file to define the metadata information.</p><p>This notebook mainly contains the below sections:</p><ul><li>Define a training config with <code>JSON</code> or <code>YAML</code> format</li><li>Execute training based on bundle scripts and configs</li><li>Hybrid programming with config and python code</li></ul><p>You can find the usage examples of MONAI bundle key features and syntax in this tutorial, like:</p><ul><li>Instantiate a python object from a dictionary config with <code>_target_</code> indicating class or function name or module path.</li><li>Execute python expression from a string config with the <code>$</code> syntax.</li><li>Refer to other python object with the <code>@</code> syntax.</li><li>Require other independent config items to execute or instantiate first with the <code>_requires_</code> syntax.</li><li>Macro text replacement with the <code>%</code> syntax to simplify the config content.</li><li>Leverage the <code>_disabled_</code> syntax to tune or debug different components.</li><li>Override config content at runtime.</li><li>Hybrid programming with config and python code.</li></ul><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Sure, updated in the latest commit.

Thanks.